console/src/app/modules/user-grants/user-grants.component.ts

```diff
@@ -61,6 +61,7 @@ export class UserGrantsComponent implements OnInit, AfterViewInit {
   public loadedId: string = '';
   public loadedProjectId: string = '';
   public grantToEdit: string = '';
+  public showAddButton: boolean = false;
```

Comment:
```suggestion
```

Reply:
we have to initialize the routerLink with undefined in order to check it in the html.
```
public routerLink: any = undefined;
```